Registry/RegistryParse.py

```diff
@@ -288,19 +346,19 @@ def hive_name(self):
     def first_hbin_offset(self):
         """
         Get the buffer offset of the first HBINBlock as an unsigned integer.
-        Note: always returns 0x1000 nothing else is known.
+        Note: always returns 0x1000, nothing else is possible.
         """
         return 0x1000
 
-    def last_hbin_offset(self):
```

Comment:
I agree with @NiKiZe here. Until we bump the major version to 2.x, I'd like to avoid breaking changes to the API. So, warning that `last_hbin_offset()` is deprecated and returning a sane (but potentially buggy) value is preferable.

We can start planning for 2.x now, and begin advertising the breaking changes. I'll open an issue for this to track it.